yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -754,10 +755,11 @@ def add(meta_list, info_list=None):
         add(('description', 'synopsis'), 'description')
         add(('purl', 'comment'), 'webpage_url')
         add('track', 'track_number')
-        add('artist', ('artist', 'creator', 'uploader', 'uploader_id'))
-        add('genre')
+        add('artist', ('artists', 'creator', 'uploader', 'uploader_id'))
```

Comment:
I found a new issue. These use-cases will break. We need a way to handle it

https://github.com/yt-dlp/yt-dlp/blob/85b33f5c163f60dbd089a6b9bc2ba1366d3ddf93/README.md#L1781-L1785

I'm not sure what's the best soln, but this should work:

```suggestion
        add('artist', ('artists', 'artist', 'creator', 'uploader', 'uploader_id'))
```

Opinion?